blakserv/ccode.c

```diff
@@ -17,6 +17,8 @@
 */
 
 #include "blakserv.h"
+#define FMT_HEADER_ONLY

```

Comment:
If we use FMT_HEADER_ONLY, then is it possible that we don't need the .cc files, and we don't need to build fmtlib.lib?